src/mobility/components/PayWithBonusPointsCheckbox.tsx

```diff
@@ -51,7 +51,7 @@ export const PayWithBonusPointsCheckbox = ({
       <GenericClickableSectionItem
         active={isChecked}
         onPress={onPress}
-        onAccessibilityTap={onPress}
+        onAccessibilityTap={() => !disabled && onPress()}
```

Comment:
I don't think we need `onAccessibilityTap` on something that already is pressable. We haven't used it anywhere else in the app until now.